### ISSUE TITLE
RollingFileAppender not call getFileHeader() when rolling

### ISSUE THREAD

**Issue by Se7enShawn** [qos-ch/logback#865]
From version 1.5.7 the getFileHeader() method in HTMLLayoutBase is called only at startup and is no longer called in files created subsequently.

```
<appender name="LOG" class="ch.qos.logback.core.rolling.RollingFileAppender">
	<File>info.html</File>
	<encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
		<layout class="MyHTMLLayout">
			<pattern>%date%thread%level%logger%msg</pattern>
			<cssBuilder class="MyCssBuilder" />
		</layout>
	</encoder>
	<filter class="ch.qos.logback.classic.filter.ThresholdFilter">
		<level>info</level>
	</filter>
	<rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
		<fileNamePattern>info.%d{yyyy-MM-dd_HH-mm}.html</fileNamePattern>
	</rollingPolicy>
</appender>
```

**Comment by johannesherr** [comment]
Could be a duplicate of https://github.com/qos-ch/logback/issues/857?

**Comment by ceki** [comment]
@johannesherr  Thank you for noting this. Issue was fixed in version 1.5.9. Marking issue as duplicate and closing.
